builds/e2e/templates/e2e-run.yaml

```diff
@@ -0,0 +1,54 @@
+steps:
+- pwsh: |
+    $testFile = '$(binDir)/Microsoft.Azure.Devices.Edge.Test.dll'
+
+    if ($IsWindows)
+    {
+      # The windows installer script doesn't update the PATH in the current
+      # terminal session, so add `iotedge` and `docker` CLIs manually
+      $env:Path =
+        $env:Path,
+        $(Join-Path $env:ProgramFiles 'iotedge'),
+        $(Join-Path $env:ProgramFiles 'iotedge-moby') -join ';'
+
+      dotnet vstest $testFile --logger:trx
```

Comment:
will it be difficult to see which test is failed since all tests are running in 1 single step in vsts pipeline? is it possible to separate each tests as steps in vsts pipeline as before?